app/src/main/res/values/strings.xml

```diff
@@ -426,11 +426,6 @@
     <string name="wallpaper_updated_snackbar_message">Wallpaper updated!</string>
     <!-- Snackbar label for action to view selected wallpaper -->
     <string name="wallpaper_updated_snackbar_action">View</string>
-    <!-- Label for switch which toggles the "tap-to-switch" behavior on home screen logo -->
-    <string name="wallpaper_tap_to_change_switch_label_1">Change wallpaper by tapping Firefox homepage logo</string>
-    <!-- This is the accessibility content description for the wallpapers functionality. Users are
-    able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
-    <string name="wallpaper_logo_content_description">Firefox logo - change the wallpaper, button</string>
```

Comment:
Ah, I should've specified. We actually have a [special process](https://github.com/mozilla-mobile/fenix/wiki/Removing-strings) for removing strings. Please add these back in and tag them as described on that wiki page. Thanks for patience in getting this ship-shape!

Reply:
Thanks for letting me know about the process. I have done the changes as mentioned there.